homeassistant/components/fireservicerota/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "fireservicerota",
```

Comment:
We normally separate words with underscore.

Reply:
It's a Brand name, like smartthings, netgear, zoneminder. If you really want me to change it no problem but it's kinda rigorous in this stage. Let me know.

Reply:
If that's how it's written in the brand name, then it's good to keep.